### PR TITLE
Bundled profiles and browser tests

### LINKED ISSUE #39: Add in-browser tests
# Overview

Correct me if I'm wrong but for now build artifacts for browser is not tested at all.

# Tasks

- [ ] setup zombiejs for tests
- [ ] write tests

### LINKED ISSUE #43: Include schemas into webpack build
# Overview

For now in browsers it works only in remote mode (using calls to schemas registry). We could improve speed a lot including builtin schemas into webpack build. It's possible using `System.js` or dynamic `require`:
- https://webpack.github.io/docs/context.html
- https://github.com/kentcdodds/es6-todomvc/blob/da6fb78322c8849096d200bc357e53f840c5bf17/src/graph/index.js#L49-L53 (see also webpack config)
- https://github.com/systemjs/systemjs

# Tasks
- [ ] find out how to achieve the goal
- [ ] implement 
- [ ] tests

### LINKED ISSUE #45: Disable profile stuff by default
Profile is used quite rarely IME and it results in automatic http calls (which is annoying if working locally offline).

Could we consider having profile support off by default?

### PR DESCRIPTION
**Pull request by dumyan**
- fixes #43
- fixes #45
- fixes #39 
- fixes [partially] #54
---
Hey @roll, please review the current work when you are able

Webpack is used for bundling profiles because it was the cleanest solution, SystemJS required hacking the json loader.

Tests are currently bundled with the module with webpack, so they can be easily shared with the node tests. Currently they can be seen by opening `test/browser/index.html`, I had an issue using phantomjs, specifically this one https://github.com/ariya/phantomjs/issues/13895. Also porting the tests is still in progress.

Also, addresses #54

### CODE REVIEW

**Comment by pwalsh** [comment]
why are we using phantoms for tests??

File: src/profiles.js [hunk 1]
```diff
@@ -3,11 +3,10 @@ import _ from 'lodash'
 import path from 'path'
 import Utils from './utils'
 
-
 // Internal
 
-const DEFAULT_REMOTE_PATH = 'http://schemas.datapackages.org/registry.json'
-let DEFAULT_LOCAL_PATH
+const DEFAULT_REMOTE_PATH = 'https://schemas.frictionlessdata.io/registry.json'
```
Comment (roll) [inline src/profiles.js]: Is it possible to use here '//schemas...' - form without protocol?

File: src/profiles.js [hunk 2]
```diff
@@ -162,4 +164,32 @@ export default class Profiles {
...
+   */
+  _loadFile(filePath) {
+    if (Utils.isRemoteURL(filePath)) {
+      return Utils.readFileOrURL(filePath)
+    }
+
+    if (Utils.isBrowser) {
+      return new Promise((resolve, reject) => {
+        try {
+          resolve(JSON.stringify(require('./schemas/' + filePath + '.json')))
```
Comment (roll) [inline src/profiles.js]: I suppose we need a comment here to explain that this is for webpack bundling

File: src/profiles.js (hunk 1 above)
Comment (dumyan) [inline src/profiles.js]: It's possible with modification of the `Utils._isRemoteURL` to include `//` and while in node this works fine, I'm getting this error when testing in the browser:

```
SyntaxError
      at XMLHttpRequest.open (node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:458:15)
      at about:blank:49369:12
      at self.fetch (about:blank:49346:13)
      at _readURL (about:blank:25508:17)
      at Function.readFileOrURL (about:blank:25534:19)
      at Profiles._loadFile (about:blank:25400:33)
      at Profiles._loadRegistry (about:blank:25286:20)
      at about:blank:25206:14
      at new Profiles (about:blank:25205:13)
      at _callee$ (test/browser/profiles.js:18:30)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:64:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:355:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:116:21)
      at step (test/browser/profiles.js:19:191)
      at test/browser/profiles.js:19:437
      at Context.<anonymous> (test/browser/profiles.js:19:99)

```

**Pull request merged**